spec/javascript/components/StateTileTest.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import StateTile from '../../../app/javascript/react/components/StateTile'
+import testHelper from '../testHelper'
+import {BrowserRouter} from "react-router-dom"
+
+
+describe('StateTile', () => {
+  let wrapper
+
+  beforeEach(() => {
+    jasmineEnzyme();
```

Comment:
Check whether your testHelper has beforeEach(() => jasmineEnzyme()) -- if so, remove it here; if not, consider adding it there instead.